src/components/home/category/HomeCategory.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+// css
+import './HomeCategory.css'
+
+// static json
+import { featuredCategory } from '../../../static/home/featuredCategory'
```

Comment:
Just a suggestion, you can use the absolute path instead of the relative path as you have a deeply nested hierarchy so that it will be easier to import.

Reply:
made the change as suggested